### PR TITLE
Feature/86cw1rnkr page for summary transcript will be sent

### PR DESCRIPTION
**Pull request by ParulGarg20**
Created the page for "your summary and transcript will be sent.." and a new component for the recipients list displayed on the page.
[https://app.clickup.com/t/86cw1rnkr](url)

### CODE REVIEW

File: frontend/src/components/sendRecipientsList.svelte [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    Author: Parul Garg
+    Last Modified: 10/08/2024
+-->
+
+<script>
+    import { ContactsStore } from "../stores/contacts-store"
+
+</script>
+  
+<div class="flex pb-8 flex-wrap justify-center self-center max-w-xl gap-2">
```
Comment (aleu0007) [inline frontend/src/components/sendRecipientsList.svelte:14]: In the future make sure to comment what the styling was for. eg which section

**Review by aleu0007 — CHANGES_REQUESTED** [review]
Please add comments for html elements

File: frontend/src/components/sendRecipientsList.svelte (hunk 1 above)
Comment (ParulGarg20) [reply frontend/src/components/sendRecipientsList.svelte:14]: Done!

**Review by aleu0007 — APPROVED** [review]
Great work !

**Pull request merged**